utils_bash.sh

```diff
@@ -0,0 +1,153 @@
+#!/usr/bin/env bash
```

Comment:
what is it the utility ?

error executing script : 
```
   ~/git_ecocode/ecoCode-java   PR_62 ✔  ./utils_bash.sh                                                                                                                   ✘ 130 
./utils_bash.sh: line 16: declare: -A: invalid option
declare: usage: declare [-afFirtx] [-p] [name[=value] ...]
Usage utils_bash.sh [OPTIONS] COMMAND
Commands:
test                Run unit tests
lint                Linter the application's bash code
doc                 Generate API documentation
mkdocs              Start the mkdocs server
Options:
-h, --help          Display help
-v, --verbose       Make the command more talkative
    

```

Reply:
Same comment as for the toolbox.sh script.

Reply:
seen together. ok.